connectors/slack/fusebit.json

```diff
@@ -0,0 +1,24 @@
+{
+  "security": {
+    "authentication": "optional",
+    "authorization": [
+      {
+        "action": "function:execute",
+        "resource": "/account/{{accountId}}/subscription/{{subscriptionId}}/boundary/{{boundaryId}}/function/{{functionId}}/"
+      }
+    ],
+    "functionPermissions": {
+      "allow": [
+        {
+          "action": "storage:*",
+          "resource": "/account/{{accountId}}/subscription/{{subscriptionId}}/storage"
```

Comment:
Trailing slash, perhaps?

Reply:
Does the connector need access to *all storage on the subscription*? (See a similar comment on the addon)

Reply:
It doesn't, it only needs permission to create the storage for its connector instance... however we don't know the boundary where that's going to live at this stage.